gno.land/pkg/sdk/vm/common_test.go

```diff
@@ -55,15 +49,38 @@ func _setupTestEnv(cacheStdlibs bool) testEnv {
 	mcw := ms.MultiCacheWrap()
 	vmk.Initialize(log.NewNoopLogger(), mcw)
 	stdlibCtx := vmk.MakeGnoTransactionStore(ctx.WithMultiStore(mcw))
-	stdlibsDir := filepath.Join("..", "..", "..", "..", "gnovm", "stdlibs")
-	if cacheStdlibs {
-		vmk.LoadStdlibCached(stdlibCtx, stdlibsDir)
-	} else {
-		vmk.LoadStdlib(stdlibCtx, stdlibsDir)
-	}
+	loadStdlibs(vmk.gnoStore)
 	vmk.CommitGnoTransactionStore(stdlibCtx)
 	mcw.MultiWrite()
 	vmh := NewHandler(vmk)
 
 	return testEnv{ctx: ctx, vmk: vmk, bank: bank, acck: acck, vmh: vmh}
 }
+
+func loadStdlibs(store gno.Store) {
+	stdlibInitList := stdlibs.InitOrder()
+	for _, lib := range stdlibInitList {
+		if lib == "testing" {
+			// XXX: testing is skipped for now while it uses testing-only packages
+			// like fmt and encoding/json
+			continue
+		}
+		loadStdlibPackage(lib, store)
+	}
+}
+
+func loadStdlibPackage(pkgPath string, store gno.Store) {
+	memPkg := stdlibs.EmbeddedMemPackage(pkgPath)
+	if memPkg == nil || memPkg.IsEmpty() {
+		// no gno files are present
+		panic(fmt.Sprintf("failed loading stdlib %q: not a valid MemPackage", pkgPath))
```

Comment:
it happens if you pass an invalid pkgpath to `stdlibs.EmbeddedMemPackage`
I panic because I didn't want to change `setupTestEnv` interface, the previous implem using cached stdlib could also panic